### PR TITLE
Add description of GraphQL Server in Azure Marketplace doc

### PR DESCRIPTION
**Pull request by kota2and3kan**
This PR updates the following two things in the Azure Marketplace doc.

1. Add description of Scalar DB GraphQL Server since it is available now.
1. Update the structure of the example part according to the following comment (update the structure same as AWS doc).
    * https://github.com/scalar-labs/scalar-kubernetes/pull/184#discussion_r947446402

Please take a look!

### CODE REVIEW

File: docs/AzureMarketplaceGuide.md [hunk 1]
```diff
@@ -55,77 +57,87 @@ Please refer to the [Azure Container Registry documentation](https://docs.micros
...
+          image:
+            repository: "example.azurecr.io/scalarinc/scalardb-server"
+            tag: "3.5.2"
+        ```
+      * Scalar DB GraphQL Server (scalardb-graphql-custom-values.yaml)
+        ```yaml
+        image:
+          repository: "example.azurecr.io/scalarinc/scalardb-graphql"
+          tag: "3.6.0"
+        ```
```
Comment (kota2and3kan) [inline docs/AzureMarketplaceGuide.md:80]: This is an added description of Scalar DB GraphQL Server.

File: docs/AzureMarketplaceGuide.md [hunk 2]
```diff
@@ -55,77 +57,87 @@ Please refer to the [Azure Container Registry documentation](https://docs.micros
...
-         ```
+   * Scalar DB Examples
+      * Scalar DB Server
+        ```console
+        helm install scalardb scalar-labs/scalardb -f ./scalardb-custom-values.yaml
+        ```
+      * Scalar DB GraphQL Server
+        ```console
+        helm install scalardb-graphql scalar-labs/scalardb-graphql -f scalardb-graphql-custom-values.yaml
+        ```
```
Comment (kota2and3kan) [inline docs/AzureMarketplaceGuide.md:127]: This is an added description of Scalar DB GraphQL Server.

File: docs/AzureMarketplaceGuide.md [hunk 3]
```diff
@@ -150,72 +162,79 @@ Please refer to the [Azure Container Registry documentation](https://docs.micros
...
+            - name: "reg-acr-secrets"
+        ```
+      * Scalar DB GraphQL Server (scalardb-graphql-custom-values.yaml)
+        ```yaml
+        image:
+          repository: "example.azurecr.io/scalarinc/scalardb-graphql"
+          tag: "3.6.0"
+        imagePullSecrets:
+          - name: "reg-acr-secrets"
+        ```
```
Comment (kota2and3kan) [inline docs/AzureMarketplaceGuide.md:191]: This is an added description of Scalar DB GraphQL Server.

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**